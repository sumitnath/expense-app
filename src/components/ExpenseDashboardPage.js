import React from 'react';
//import { BrowserRouter,Route,Switch,Link,NavLink } from "react-router-dom";
import ExpenseList from './ExpenseList';
import ExpenseListFilter from './ExpenseListFilter';

const ExpenseDashboardPage = () => (
  <div>
    This is from my dashboard component
    <ExpenseListFilter/>
    <ExpenseList/>
  </div>
);


  export default ExpenseDashboardPage;