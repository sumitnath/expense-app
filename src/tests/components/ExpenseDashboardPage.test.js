import React from "react";
import ExpenseDashboardPage from "../../components/ExpenseDashboardPage";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

test('should render Dashboard page correctly', () => {
    const wrapper = shallow( < ExpenseDashboardPage / > )
    expect(toJson(wrapper)).toMatchSnapshot()
})