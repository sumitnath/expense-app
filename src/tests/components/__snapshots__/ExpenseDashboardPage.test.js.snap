// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dashboard page correctly 1`] = `
<div>
  This is from my dashboard component
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
